### DISCUSSION TITLE
Tool not found: studio

### DISCUSSION THREAD

**Discussion by cmcollin41** [sanity-io/sanity#3828] [Sanity Studio Support] [answered]
I'm using NextJS and changing from v2 to v3 and when I load my studio at `pages/studio/[[...index]]` i get `Tool not found: studio`.

**Comment by valse** [comment, marked as answer]
Hi, you need to add the `basePath: "/studio"` option on your sanity config

**Reply by cmcollin41** [reply]
Awesome! Thank you for the reply.
